### PR TITLE
#12:Create and style for comment component

### PR DESCRIPTION
**Pull request by ngtrongnguyen**
Ảnh chụp Màn hình 2023-06-09 lúc 14 20 29

### CODE REVIEW

File: src/components/comment-card.html [hunk 1]
```diff
@@ -0,0 +1,16 @@
+<div class="comment">
+  <article class="comment-card">
+      <figure class="comment-card-reviewers__img ">
+          <img src="../assets/img/Ellipse 31.png" alt="" class="comment-imge">
```
Comment (Lanng21) [inline src/components/comment-card.html:4]: alt?

File: src/components/comment-card.html [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+      </figure>
+      <blockquote class="comment-card-reviewers__comment ">
+          {{comment}}
+      </blockquote>
+      <img src="../assets/img/Union.png" alt="" class="comment-union">
```
Comment (Lanng21) [inline src/components/comment-card.html:9]: alt?

File: src/stylesheets/components/_comment.scss [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+                        } 
+                    }
+                    .comment-union{
+                        display: block;
+                        float: right;
+                        margin: 20px;
+                    }
+                }
+            }
+            .comment-elementbg{
```
Comment (Lanng21) [inline src/stylesheets/components/_comment.scss:56]: remove if u dont style

File: src/stylesheets/components/_comment.scss [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+
+            }
+
+            .comment-card-reviewers__comment {
```
Comment (Lanng21) [inline src/stylesheets/components/_comment.scss:60]: same

**Comment by Lanng21** [comment]
Pls, update the screenshot again.

**Review by haunguyen15-tt — APPROVED** [review]

**Comment by Lanng21** [comment]
Resolve conflicts

**Review by Lanng21 — APPROVED** [review]

**Pull request merged**